cli/internal/runsummary/task_summary.go

```diff
@@ -51,11 +51,12 @@ func (ts *TaskSummary) cleanForSinglePackage() {
 	for i, dependent := range ts.Dependents {
 		dependents[i] = util.StripPackageName(dependent)
 	}
+	taskID := util.StripPackageName(ts.TaskID)
```

Comment:
Can you keep this as `task` though, it's not a taskId anymore

```suggestion
	task := util.StripPackageName(ts.TaskID)
```